packages/passport/sdk/src/Passport.ts

```diff
@@ -134,35 +135,74 @@ export class Passport {
    * `connectImx` instead.
    */
   public async connectImxSilent(): Promise<IMXProvider | null> {
-    return this.passportImxProviderFactory.getProviderSilent();
+    const flow = trackFlow('passport', 'connectImxSilent');
+
+    try {
+      flow.addEvent('startConnectImxSilent');
```

Comment:
The `start` event seems to be triggered when `trackFlow` is called (see [here](https://github.com/immutable/ts-immutable-sdk/blob/d4aa5e257dbdf681cdd241decbe689f891ca4476/packages/internal/metrics/src/flow.ts#L104)), so we should be able to omit all of these `flow.addEvent('startXXXXX');` calls

Reply:
Ahh good catch, thanks!